File: common/src/main/java/com/itsmtools/common/dictionary/service/spec/ContractorService.java
package com.itsmtools.common.dictionary.service.spec;


import com.itsmtools.common.dictionary.model.Contractor;

import java.util.List;

public interface ContractorService extends BaseService{
    Contractor item(Integer id);
    void save(Contractor entity);
    void update(Contractor entity);
    void delete(Integer id);
    List<Contractor> list();
}

File: common/src/main/resources/application/module/common/dictionary/data/service/contractor-service.js
angular.module("common.dictionary.data")
  .factory("contractorService", function ($http) {
    return {
      item: function (id) {
      },

      list: function (start, limit) {
        return $http.get("/contractor/list", {}).then(function (response) {
          return response.data;
        });
      },

      update: function (record) {
      },

      new: function (record) {
      },

      delete: function (id) {
      }
    }
  });